addons/l10n_es_edi_facturae/models/account_move.py

```diff
@@ -136,6 +136,11 @@ def _l10n_es_edi_facturae_get_refunded_invoices(self):
     def _l10n_es_edi_facturae_get_corrective_data(self):
         self.ensure_one()
         if self.move_type.endswith('refund'):
+            if not self.reversed_entry_id:
+                raise UserError(_("The credit note/refund appears to have been issued manually. For the purpose of "
+                                  "generating a Facturae document, it's necessary that the credit note/refund is created "
+                                  "directly from the associated invoice/bill."))
+
```

Comment:
pot ?

Reply:
Done!
Thanks for taking the time to look into this.